.changeset/heavy-schools-argue.md

```diff
@@ -1,5 +1,5 @@
 ---
-'@talend/react-forms': major
+'@talend/react-forms': patch
```

Comment:
A minor version looks more appropriate here. We don't fix a bug, we don't break an existing feature, we just add a new thing 😬 

```suggestion
'@talend/react-forms': minor
```